packages/tools/pie-icons-configs/configs.ts

```diff
@@ -0,0 +1,129 @@
+// TODO: DSW-1025 - Use this file instead of the JS file
```

Comment:
Just commenting that i think this is a fantastic improvement! I'd be keen for us to use TS for these kinds of build scripts and packages going forward!